blog/releases/announcing-knative-v0-17-release.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Announcing the Knative v0.17 release"
+linkTitle: "Version 0.17 release"
+date: 2020-08-24
+description: "The Knative v0.17 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+## Announcing Knative v0.17 Release
+
+As Knative maintains its predictable release cadence, it's time to cut another stable release across components. Be aware for the next release we are planning to make Kubernetes 1.17 the minimum version. Serving continues their journey with test and benchmark optimizations on the already v1 API version. For Eventing now ContainerSource and PingSource are available as v1beta1.
+
+Remember to check [the upgrade docs](https://knative.dev/docs/install/upgrade-installation/) for any upgrades concerns applicable to your current version before you upgrade to the new version v0.17
+
+
+### [Serving v0.17.1](https://github.com/knative/serving/releases/tag/v0.17.1)
+
+**InitialScale annotation to control the initial deployment size**
+There is a new annotation that can be used to control the number of pods that are initially deployed when new Revisions are rolled out.
+
+**net-contour and net-kourier have moved to Beta**
+In addition to net-istio, we now have three networking layers that we classify as (at least) Beta.
+
+**Kubernetes minimum version has NOT changed**
+It remains **1.16**, but we may bump to 1.17 in the coming release depending on its availability for qualification testing (per our release principles).
```

Comment:
We shouldn't document what may or may not change in the future. See https://developers.google.com/style/future

I think we should remove all mentions of this 1.17 unless there's somewhere that we previously stated the Kubernetes version will update with each Knative version (I assume we don't do this). Otherwise, no need IMHO to call out what _didn't_ change :woman_shrugging:

Reply:
I can remove, but I copy this verbatim from the Service [release notes](https://github.com/knative/serving/releases/tag/v0.17.2)

I guess it was more a was to give heads up that imminent changes are coming in near term releases.